src/components/RedocStandalone.tsx

```diff
@@ -14,6 +14,8 @@ export interface RedocStandaloneProps {
   onLoaded?: (e?: Error) => any;
 }
 
+declare let __webpack_nonce__: string;
```

Comment:
Could you please move it to the custom.d.ts file?

Reply:
Actually, this is already defined in the index.d.ts of the Webpack package.